core/README.dbconfig

```diff
@@ -84,7 +84,7 @@ postgresql
   * update from 13.2:
     * db_version is already 2002. It detects, nothing to do.
   * update dbconfig already configured
-    * ?
+
 
 mysql
```

Comment:
Shouldn't we already remove all mysql / sqlite ?

Reply:
I decided against it, as this PR is not about removing MySQL/Sqlite.